packages/element/src/serialize.js

```diff
@@ -494,10 +501,22 @@ export function renderComponent(
 	context,
 	legacyContext = {}
 ) {
-	const instance = new Component( props, legacyContext );
+	const instance = new /** @type {import('react').ComponentClass} */ ( Component )(
+		props,
+		legacyContext
+	);
 
-	if ( typeof instance.getChildContext === 'function' ) {
-		Object.assign( legacyContext, instance.getChildContext() );
+	if (
+		typeof (
+			// Ignore reason: Current prettier reformats parens and mangles type assertion
```

Comment:
Maybe already known based off "current", but I believe this may be improved in the recently-released Prettier 2.0

https://prettier.io/blog/2020/03/21/2.0.0.html#ultimate-fix-for-closure-style-type-casts-7791httpsgithubcomprettierprettierpull7791-by-thorn0httpsgithubcomthorn0-7011httpsgithubcomprettierprettierpull7011-by-evilebottnawihttpsgithubcomevilebottnawi

Reply:
Yes! That fix sounds like just what we need.